process faults {3: ({0, 4}, 2311411807)}
network faults {}
{6: ("29157E29FCE7D4659053E807CB3722C462036BC00C0889DD551B09913F5C98DC", {1, 6, 3, 2, 4, 0, 5}), 7: ("6A04D3249BE81D729D3C55D9A6745A6D24311AEC467C36BEF1DD6B2DA60D5CE7", {1, 3, 4, 0, 6, 2, 5}), 8: ("7BB13AD4B2D648090B0D9B3C723E7CA3B11FFE145781D923A5718F20C39EC906", {2, 1, 0, 5, 3, 4, 6}), 9: ("78DCDCC222DB52C94CCE3827AC34E82418D406F016E064B93E023EC7D38F7C09", {6, 4, 0, 1, 5, 2, 3}), 10: ("26697EEDC209297176679E26BFCAE5F7386B58CDA0AB5DF85E36182BB7F6CB1E", {5, 4, 0, 3, 1, 6, 2}), 5: ("07C9F393F267F4988F8ABA1973101A61B88D0F49962A7B7126421CA91DA894E2", {3, 4, 2, 6, 0, 5, 1})}
7
reason: all committed
done!
